src/components/pages/lessons/LessonInfo.tsx

```diff
@@ -5,46 +5,45 @@ import {isEmpty, get} from 'lodash'
 import Markdown from 'react-markdown'
 import useCopyToClipboard from 'react-use/lib/useCopyToClipboard'
 import Eggo from '../../../components/images/eggo.svg'
-import {LessonResource} from 'types'
+import {useNextUpData} from 'hooks/use-next-up-data'
 
 type NextUpProps = {
   currentLessonSlug: string
-  data: {
-    list: {
-      lessons: LessonResource[]
-    }
-  }
+  url: string
 }
 
-const NextUp: FunctionComponent<NextUpProps> = ({data, currentLessonSlug}) => {
-  return data ? (
+const NextUp: FunctionComponent<NextUpProps> = ({url, currentLessonSlug}) => {
+  const {nextUpData} = useNextUpData(url)
```

Comment:
moving network calls into the components they are used in so we aren't loading everything up front on load by default